### PR TITLE
Improve run-image's boot.sh.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: system/extensions/host/run-image.toit [hunk 1]
```diff
@@ -344,13 +344,15 @@ add-image path/string existing-uuids/Set --run-boot/bool --run-critical/bool ->
   file-uuid/uuid.Uuid? := uuid.parse last-segment --on-error=: null
 
   if file-uuid and existing-uuids.contains file-uuid:
+    print "Already in flash: $file-uuid"
```
Comment (kasperl) [inline system/extensions/host/run-image.toit:347]: Add TODO for getting rid of this again.

File: system/extensions/host/run-image.toit [hunk 2]
```diff
@@ -344,13 +344,15 @@ add-image path/string existing-uuids/Set --run-boot/bool --run-critical/bool ->
...
     // Already in the flash.
     return
 
   image-data := file.read-content path
   writer := containers.ContainerImageWriter image-data.size
   writer.write image-data
   writer.commit --run-boot=run-boot --run-critical
+  print "Installed in flash: $file-uuid"
```
Comment (kasperl) [inline system/extensions/host/run-image.toit:355]: Add TODO for getting rid of this again.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: system/extensions/host/run-image.toit (hunk 1 above)
Comment (floitsch) [reply system/extensions/host/run-image.toit]: removed it.
done.

File: system/extensions/host/run-image.toit (hunk 2 above)
Comment (floitsch) [reply system/extensions/host/run-image.toit]: removed it.
done.

**Pull request merged**